.sync.yml

```diff
@@ -2,11 +2,11 @@
 .github/workflows/ci.yaml:
   acceptance_matrix:
     set:
-      - el7
+      - ---el7
```

Comment:
I don't think this is necessary as EL7 is being removed from `metadata.json`.

Reply:
It is due to the PDK template I use: https://github.com/treydock/pdk-templates/blob/5a59870af386c8ab7a5e766e8e8b1534dd459fc7/config_defaults.yml#L26.

Unfortunately I still run RHEL7 so haven't got rid of it in all my modules just yet.